src/app/component/eco-news/components/create-news/create-news.component.scss

```diff
@@ -58,15 +58,16 @@ form {
     display: flex;
     flex-direction: column;
     justify-content: space-between;
-    height: 380px;
 }
 
-.left-form-column input {
+.left-form-column input, .left-form-column textarea {
```

Comment:
Please, if you have more than one class, put them into separate rows

Reply:
fixed